README.md

```diff
@@ -1,23 +1,108 @@
 <!---
-  Copyright 2023 SECO Mind Srl
+  Copyright 2023-2024 SECO Mind Srl
   SPDX-License-Identifier: Apache-2.0
 -->
 
-# EdgehogDeviceForwarder
+# Edgehog Device Forwarder
 
-To start your Phoenix server:
+Proxy server to facilitate communication with Edgehog devices.
+It handles HTTP messages and WebSocket connections between the device and the user client.
 
-  * Run `mix setup` to install and setup dependencies
-  * Start Phoenix endpoint with `mix phx.server` or inside IEx with `iex -S mix phx.server`
+It's only been tested with simple WebSocket connections such as
+[ttyd](https://github.com/tsl0922/ttyd), however it should handle arbitrary requests
+as long as you keep in mind the [limitations](#known-issues-and-limitations).
 
-Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
+## Running the Forwarder locally
 
-Ready to run in production? Please [check our deployment guides](https://hexdocs.pm/phoenix/deployment.html).
+### Start the server
 
-## Learn more
+The first component to be available should be this server,
+as both the device and the client will connect to it.
 
-  * Official website: https://www.phoenixframework.org/
-  * Guides: https://hexdocs.pm/phoenix/overview.html
-  * Docs: https://hexdocs.pm/phoenix
-  * Forum: https://elixirforum.com/c/phoenix-forum
-  * Source: https://github.com/phoenixframework/phoenix
+#### Manual installation
+
+Run the following command to clone the current repository and navigate to it
+
+```bash
+git clone https://github.com/edgehog-device-manager/edgehog_device_forwarder && \
+cd edgehog_device_forwarder
+```
+
+> At the time of writing, not all the relevant code has been merged into the main branch.
+> Use the following command command instead
+>
+> ```bash
+> git clone https://github.com/noaccOS/edgehog_device_forwarder \
+> -b feat/fallback-controller && cd edgehog_device_forwarder
+> ```
+
+You will need to install Elixir 1.15 with OTP 26.
+
+> If you use [`asdf`](https://asdf-vm.com), you can use it to install our officially supported
```

Comment:
Maybe better make it impersonal, if you think this makes sense (I might not fully understand the context).

... ,you can use it to install ~~our~~ _the Edgehog_ officially supported

Reply:
This section has been reworked as with #14 (not yet merged) we now include `.tool-versions` in the forwarder.